modules/apigee_edge_debug/src/SDKConnector.php

```diff
@@ -20,22 +20,19 @@
 
 namespace Drupal\apigee_edge_debug;
 
+use Apigee\Edge\ClientInterface;
 use Drupal\apigee_edge\SDKConnector as OriginalSDKConnector;
 use Drupal\apigee_edge\SDKConnectorInterface;
-use Drupal\Core\Config\ConfigFactoryInterface;
-use Drupal\Core\Entity\EntityTypeManagerInterface;
-use Drupal\Core\Extension\InfoParserInterface;
-use Drupal\Core\Extension\ModuleHandlerInterface;
-use Drupal\Core\Http\ClientFactory;
-use Drupal\key\KeyRepositoryInterface;
+use Drupal\key\KeyInterface;
+use Http\Message\Authentication;
 
 /**
  * Service decorator for SDKConnector.
  */
-class SDKConnector extends OriginalSDKConnector implements SDKConnectorInterface {
+final class SDKConnector implements SDKConnectorInterface {
```

Comment:
This will force more re-work in the monetization module. Many classes in this module are being marked as final which keeps them from being extended.

Reply:
I checked the Monetization module before I created this PR and I haven't found anything that this change would affect or make it impossible to rewrite with encapsulation which is a better design.
@Jaesin I may have missed something, could you give me an example that you see problematic?

Reply:
Double check it, I used what it is currently in the 8.x-1.x branch. The MINT module only depends on the `SDKConnectorInterface` and the method that it provides, it does not depend on a concrete implementation in the SDKConnector. :tada: The MINT module does not use `\Drupal\apigee_edge\SDKConnectorInterface::buildClient()` method which has been removed.

@Jaesin let me know if I missed something.

Reply:
```
Testing started at 15:02 ...
.../vendor/phpunit/phpunit/phpunit --configuration /C3/www/apigee/dev-mint-d8/phpunit.xml .../modules/contrib/apigee_m10n --teamcity
PHPUnit 6.5.13 by Sebastian Bergmann and contributors.

Testing .../modules/contrib/apigee_m10n

Error : Call to private method Drupal\apigee_edge\SDKConnector::buildClient() from context 'Drupal\Tests\apigee_m10n\Kernel\TestFrameworkKernelTest'
 .../modules/contrib/apigee_m10n/tests/src/Kernel/TestFrameworkKernelTest.php:58
 

Error : Call to private method Drupal\apigee_edge\SDKConnector::buildClient() from context 'Drupal\Tests\apigee_m10n\Kernel\TestFrameworkKernelTest'
 .../modules/contrib/apigee_m10n/tests/src/Kernel/TestFrameworkKernelTest.php:76
 


Time: 9.88 minutes, Memory: 24.00MB


ERRORS!
Tests: 122, Assertions: 1175, Errors: 2.
```